src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleMethodSymbol.vb

```diff
@@ -146,5 +146,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Return other Is Me OrElse
                 (other IsNot Nothing AndAlso TypeSymbol.Equals(Me._containingType, other._containingType, TypeCompareKind.ConsiderEverything) AndAlso Me._underlyingMethod.ConstructedFrom = other._underlyingMethod.ConstructedFrom)
         End Function
+
+        Friend Overrides ReadOnly Property HasSetsRequiredMembers As Boolean
+            Get
+                Return _underlyingMethod.HasSetsRequiredMembers
```

Comment:
>_underlyingMethod.HasSetsRequiredMembers [](http://example.com/codeflow?start=23&length=40)

Are we testing this code path?

Reply:
This specific codepath shouldn't be reachable, I'll add a throw.

Reply:
> This specific codepath shouldn't be reachable, I'll add a throw.

I think this assumption is not correct.